src/server/mod.rs

```diff
@@ -44,16 +45,13 @@ use self::stream::{SecuritySwitch, SwitchingTlsStream};
 use crate::auth;
 use crate::auth::AnonymousUser;
 use crate::metrics;
-use crate::storage;
-use crate::storage::filesystem::Filesystem;
+use crate::storage::{self, filesystem::Filesystem, Metadata};
 use session::{Session, SessionState};
 
-use rand::Rng;
-
 const DEFAULT_GREETING: &str = "Welcome to the libunftp FTP server";
 const CONTROL_CHANNEL_ID: u8 = 0;
-
 const BIND_RETRIES: u8 = 10;
+const RFC3659_TIME: &str = "%Y%m%d%H%M%S";
```

Comment:
The spec mentions milliseconds, although they are optional. Any particular reason we're skipping them here?

Reply:
I thought for FTP MVP its an overkill but I'm also happy to add them.

Reply:
As per discussion we'll keep it to seconds for now. ext2 & ext3 only supports seconds resolution for instance so to do this property one will have to check what the filesystem supports. For  now this is out of scope.